tensorflow/contrib/tensorrt/log/trt_logger.cc

```diff
@@ -14,21 +14,23 @@ limitations under the License.
 ==============================================================================*/
 
 #include "tensorflow/contrib/tensorrt/log/trt_logger.h"
-// Use TF logging for TensorRT informations
+
 #include "tensorflow/core/platform/logging.h"
 
+// Use TF logging for TensorRT informations
+
 #define _TF_LOG_DEBUG ::tensorflow::internal::LogMessage(__FILE__, __LINE__, -1)
-//------------------------------------------------------------------------------
-namespace tensorflow {
 
-//------------------------------------------------------------------------------
+namespace tensorflow {
 namespace tensorrt {
 
 void Logger::log(Severity severity, const char* msg) {
-  // suppress info-level messages
+  // Suppress info-level messages
   switch (severity) {
     case Severity::kINFO: {  // mark TRT info messages as debug!
       VLOG(-1) << msg;
+    case Severity::kINFO: {  // Mark TRT info messages as debug!
```

Comment:
merge conflicts not properly resolved :)

Reply:
@jjsjann123, yep missed that. Merge conflicts were elsewhere and didn't catch this :(